@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/animation/fn_probes.sqf

```diff
@@ -23,7 +23,7 @@ if !(player == currentPilot _heli) exitwith {};
 private _realRPM = [_heli] call fza_sfmplus_fnc_getRtrRPM;
 private _inducedFlow = [0.0, 0.0, _realRPM * 1.0];
 ([_heli, fza_ah64_sfmplusEnableWind] call fza_sfmplus_fnc_getVelocities) params ["","","","","_velModelSpace"];
-private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,-9.8]);
+private _gravity = _heli worldtomodelVisual (getpos _heli vectoradd [0,0,9.8]);
```

Comment:
Just make this a 1 value, you'll also want to do a sin and cos function using the pitch/bank of the helicopter to influence it as well.

Reply:
iv changed the value to 1, sin & cos are not necessary, I get the gravity vector by translating world to model space

Reply:
Updated